src/exported-image-egl.cpp

```diff
@@ -30,6 +30,14 @@
 
 extern "C" {
 
+__attribute__((visibility("default")))
+void
+wpe_fdo_egl_exported_image_set_destroy_notify(struct wpe_fdo_egl_exported_image* image, wpe_fdo_egl_exported_image_destroy_notify_t destroyNotify, void* destroyData)
+{
+    image->destroyNotify = destroyNotify;
+    image->destroyData = destroyData;
```

Comment:
Wouldn't it make sense that *if* a `destroyNotify` was already set previously, that the old callback is called before storing the new values? Not doing that could easily result in leaking the data passed as `destroyData`, if the callback is supposed to free it.

Reply:
Yes, that makes sense. I'll update it to do that.